content/css/concepts/display/display.md

```diff
@@ -46,6 +55,119 @@ The following values can be appplied to the `display` property:
 - `initial`
 - `inherit`
 
+## Differences between display values:
+```css
+.block-element {
+  display: block;
+}
+
+.inline-element {
+  display: inline;
+}
+
+.inline-block-element {
+  display: inline-block;
+}
+
+.hidden-element {
+  display: none;
+}
+
+.flex-container {
+  display: flex;
+}
+
+
+```
+
+## Common use cases:
+
+```css
+/* Block-level element for creating a layout with stacked elements */
+.block-element {
+  display: block;
+}
+
+/* Inline element for inline text elements */
+.inline-element {
+  display: inline;
+}
+
+/* Inline block for inline elements that need block-level properties */
+.inline-block-element {
+  display: inline-block;
+}
+
+/* Hide an element from the layout */
+.hidden-element {
+  display: none;
+}
+
+/* Flexible container for creating responsive and flexible layouts */
+.flex-container {
+  display: flex;
+}
+
+```
+
+## Overriding display values:
+
```

Comment:
```suggestion
```
You need to remove instances where you have multiple line breaks in a row. There should never be more than one line break in a row.